mobile-app/lib/ui/views/learn/challenge/challenge_viewmodel.dart

```diff
@@ -567,8 +571,8 @@ class ChallengeViewModel extends BaseViewModel {
       Challenge? currChallenge = challenge;
 
       for (ChallengeFile file in currChallenge!.files) {
-        await prefs.remove('${currChallenge.id}.${file.name}');
-        await prefs.remove('${currChallenge.id}${file.name}');
+        await prefs.remove('${currChallenge.id}.${getFullFileName(file)}');
+        await prefs.remove('${currChallenge.id}${getFullFileName(file)}');
```

Comment:
Uhhhh 🤔

Reply:
I think that might be some kind of artifact. Not sure.

Reply:
We need to check and remove it if there's no preference with that key